dashboard/app/controllers/admin_users_controller.rb

```diff
@@ -104,25 +117,35 @@ def manual_pass
   # get /admin/permissions
   def permissions_form
     search_term = params[:search_term]
-    if search_term =~ /^\d+$/
-      @user = User.find(search_term)
-    elsif search_term.present?
-      users = User.where(hashed_email: User.hash_email(search_term))
-      @user = users.first
-      if users.count > 1
-        flash[:notice] = "More than one User matches email address.  "\
-                         "Showing first result.  Matching User IDs - #{users.map(&:id).join ','}"
+    permission = params[:permission]
+    if search_term.present?
+      if search_term =~ /^\d+$/
+        @user = User.select(USER_ATTRIBUTES_FOR_VIEW).find_by(id: search_term)
+      else
+        users = User.where(hashed_email: User.hash_email(search_term))
+        @user = users.select(USER_ATTRIBUTES_FOR_VIEW).first
```

Comment:
Oh yeah, it has to be on the [ActiveRecord::Relation](https://apidock.com/rails/v4.0.2/ActiveRecord/QueryMethods/select), not on an individual `User` object.

How about something like this? Add a private method:
```ruby
def limited_users
  User.select(USER_ATTRIBUTES_FOR_VIEW)
end
```
Then, instead of `User.`..., start the queries with `limited_users.`


See also http://guides.rubyonrails.org/active_record_querying.html#selecting-specific-fields